### PR TITLE
Refactor the app for individual users and admin view for management

### PR DESCRIPTION
**Pull request by swouf**
In this PR, I removed the collaborative feature (for now) and make the app individual. In the builder, I implemented an admin view for managing (preview and download for now) the drawings of all the members.

## Working feature and improvements

- ✅ Saving basic elements is working well
- ✅ Upgrade to latest version of Excalidraw
- ✅ Admin view with thumbnails and download button for each drawing
- ✅ Player view only Excalidraw

### Work in progress

- [x] Download all button
- [x] More settings
    - [x] Export dimensions
    - [x] Default name for the drawing
    - ❌ ~Load initial drawing~ (see #37)
- ❌ ~Actions~ (see #38)
- ❌ ~Export actions, app data and settings~
- [x] Update the logo 😉 

## What is still not working (and won't be fixed in this PR)

- ❌ Library browsing and installing
- ❌ File upload/download **in** the drawing (see #24, #30, and #31)
- ❌ Collaboration (see #20)

### CODE REVIEW

File: src/modules/admin/ExcalidrawPreviewCard.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            })}
+          >
+            {getInitials(member.name)}
+          </Avatar>
+        }
+        // action={
+        //   <IconButton aria-label="settings">
+        //     <MoreVertIcon />
+        //   </IconButton>
+        // }
```
Comment (spaenleh) [inline src/modules/admin/ExcalidrawPreviewCard.tsx:117]: Remove ?

File: src/modules/common/TabPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  return (
+    <TabPanelDiv
+      role="tabpanel"
+      hidden={value !== index}
+      id={`tabpanel-${index}`}
+      aria-labelledby={`tab-${index}`}
+      {...other}
+    >
+      {/* TODO: Fix height problem with tabs. */}
```
Comment (spaenleh) [inline src/modules/common/TabPanel.tsx:26]: Is this still an issue ?

File: src/modules/main/AnalyticsView.tsx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { ANALYTICS_VIEW_CY } from '@/config/selectors';
+
+const AnalyticsView = (): JSX.Element => (
+  <div data-cy={ANALYTICS_VIEW_CY}>Player</div>
```
Comment (spaenleh) [inline src/modules/main/AnalyticsView.tsx:4]:
```suggestion
  <div data-cy={ANALYTICS_VIEW_CY}>Analytics</div>
```

File: src/modules/main/App.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+          });
+        }
+        Sentry.setContext('app-context', {
+          itemId: appContext.id,
+          name: appContext.name,
+          path: appContext.path,
+        });
+      }
+    }
+  });
```
Comment (spaenleh) [inline src/modules/main/App.tsx:42]: Nice !

**Review by spaenleh — APPROVED** [review]
This looks good to me ! I have left some comments, but I trust that you got the behaviour you want 😉

File: src/modules/admin/DownloadAllButton.tsx [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+              const d = new Date();
+              saveAs(archive, `drawings_${d.toISOString()}.zip`);
+              setIsDownloading(false);
+            });
+          })
+          .catch((reason) =>
+            // eslint-disable-next-line no-console
+            console.error(
+              t('ADMIN_VIEW.ERROR_DOWNLOAD_ALL', { reason: reason.toString() }),
+            ),
```
[the remark this answers is not in the document]
Comment (swouf) [reply src/modules/admin/DownloadAllButton.tsx]: Yes, I just updated that. I just wanted to refactor a bit the toasts first.

File: src/modules/common/TabPanel.tsx (hunk 2 above)
Comment (swouf) [reply src/modules/common/TabPanel.tsx:26]: Not sure. I don't remember fixing it but I cannot reproduce it now. It may depend on the screen resolution.

**Pull request merged**